CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+* Include spaces after commas and around operators
+
+### Methods Definitions
+
+Short form method definitions when the method body only contains 1 statement:
+
+```julia
+# Yes
+Base.stdm(v::RealArray, w::AbstractWeights, m::Real; corrected::DepBool=nothing) =
+    sqrt(varm(v, w, m, corrected=depcheck(:stdm, corrected)))
+
+# No
+function Base.stdm(v::RealArray, w::AbstractWeights, m::Real; corrected::DepBool=nothing)
+    sqrt(varm(v, w, m, corrected=depcheck(:stdm, corrected)))
+end
+```
+
+For long method definitions (and calls) our preferences is to align arguments on new lines with the first argument and avoid adding an extra newline after the last argument:
+
+```julia
+# Yes
+function Base.var(v::RealArray, w::AbstractWeights; mean=nothing,
+                  corrected::DepBool=nothing)
+    ...
+end
+
+# No
+function Base.var(
+    v::RealArray, w::AbstractWeights; mean=nothing, corrected::DepBool=nothing
+)
+    ...
+end
+
+# No
+function Base.var(
+    v::RealArray,
+    w::AbstractWeights;
+    mean=nothing,
+    corrected::DepBool=nothing
+)
+    ...
+end
+
+# Yes
+scale!(_wsum_centralize!(R, abs2, A, values(w), M, dim, true),
+       varcorrection(w, corrected))
+
+# No
+scale!(
+    _wsum_centralize!(R, abs2, A, values(w), M, dim, true),
+    varcorrection(w, corrected)
+)
+```
+
+### `return`
+
+For simple methods, we try to avoid using the `return` keyword:
+
+```julia
+# Yes
+function Base.var(v::RealArray, w::AbstractWeights; mean=nothing,
+                  corrected::DepBool=nothing)
+    corrected = depcheck(:var, corrected)
+
+    if mean == nothing
+        varm(v, w, Base.mean(v, w); corrected=corrected)
+    else
+        varm(v, w, mean; corrected=corrected)
+    end
+end
+
+# No
+function Base.var(v::RealArray, w::AbstractWeights; mean=nothing,
+                  corrected::DepBool=nothing)
+    corrected = depcheck(:var, corrected)
+
+    if mean == nothing
+        return varm(v, w, Base.mean(v, w); corrected=corrected)
+    else
+        return varm(v, w, mean; corrected=corrected)
+    end
+end
+```
+
+However, for more complicated methods it's helpful to be more explicit:
+
+```julia
+# Yes
+function counteq(a::AbstractArray, b::AbstractArray)
+    n = length(a)
+    length(b) == n || throw(DimensionMismatch("Inconsistent lengths."))
+    c = 0
+    for i = 1:n
+        @inbounds if a[i] == b[i]
+            c += 1
+        end
+    end
+    return c
+end
+
+# No
+function counteq(a::AbstractArray, b::AbstractArray)
+    n = length(a)
+    length(b) == n || throw(DimensionMismatch("Inconsistent lengths."))
+    c = 0
+    for i = 1:n
+        @inbounds if a[i] == b[i]
+            c += 1
+        end
+    end
+    c   # The `c` return is less obvious here
+end
+```
+
+### Documentation
+
+All exported functions should have a docstring associated with it.
+A docstring should start with the method signature(s) followed by a description
```

Comment:
The guidelines are also likely to be improved, so I'd prefer a conspicuously broken link than misleading outdated rules. Also I don't think the links will break that often, they are useful in many other contexts.